cmd/main_test.go

```diff
@@ -267,3 +270,83 @@ var _ = AfterSuite(func() {
 		Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Error killing the server during test teardown: %v", err))
 	}
 })
+
+type mockAuthorization struct {
+	allowed bool
+	err error
+}
+
+func (m *mockAuthorization) runAccessCheck(
+	e *echo.Echo,
+	authCl authorizationv1Client.AuthorizationV1Interface,
+	email, namespace, resourceGroup, resource, verb string,
+) (bool, error) {
+	return m.allowed, m.err
+}
+
+var _ = Describe("TestGetNamespacesWithAccess", func() {
```

Comment:
It's OK as a draft, but we'll probably want to use `DescribeTable` and parametrize the test to check different responses